hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -35,18 +36,48 @@ interface IMissingContainerResponse {
   pipelineID: string;
 }
 
+interface IContainerResponse {
+  containerID: number;
+  containerState: string;
+  unhealthySince: string;
+  expectedReplicaCount: number;
+  actualReplicaCount: number;
+  replicaDeltaCount: number;
+  reason: string;
+  keys: number;
+  pipelineID: string;
```

Comment:
```suggestion
  pipelineId: string;
```

Reply:
In Java File I can see pipelineID.

https://github.com/apache/ozone/blob/master/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/UnhealthyContainerMetadata.java

Reply:
Got it